### DISCUSSION TITLE
posts/neovim_tutor/

### DISCUSSION THREAD

**Discussion by giscus** [Parsifa1/blog#2] [Announcements]
# posts/neovim_tutor/

你应该使用这个世界上最快的编辑器（确信

https://blog.cloudti.de/posts/neovim_tutor/

**Comment by PermeationLoop** [comment]
俺的同學就和你一樣喜歡玩 Neovim，不僅給 vim 裝一大堆插件，甚至用 vim 來當 hex 編輯器，拿 vim 來當 Python 調試器卻跌得一嘴泥，老老實實滾去用 PyCharm 調試器，，，

**Reply by Parsifa1** [reply]
调试的话肯定都去用正经调试器了，而且正经调试器都是cmdline的，用习惯就好，就跟gdb一样（

**Reply by PermeationLoop** [reply]
正經調試器都是 GUI 的，不會真有人沒事去玩命令行把💢

**Comment by PermeationLoop** [comment]
## 催更博客

催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更催更

如果你需要「要寫甚麼？」的 idea 請不吝在羣裏提出，俺們很樂意助你一臂之力😋

## 俺同學的近況報告

他還是那麼喜歡玩 Neovim，最近甚至還開始用 vim 給 Rust 調試，然後被不能鼠標點弄得頭很大，甚至有點想滾回 VSC 了，一問原來是想要給「VSC 一教稱霸」的這個世界貢獻出一份反叛力量．

俺保留所有意見．
